modules/configuring/proc-installing-and-configuring-redis-cache.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-installing-and-configuring-redis-cache_{context}"]
+= Configuring the Redis cache for dynamic plugins in {product}.
+
+By default {product} does not cache plugin assets.
+You can use a Redis cache store to improve {product-very-short} performance and reliability.
+Configured plugins in {product-very-short} receive dedicated cache connections, which are powered by the Keyv redis client.
+
+.Prerequisites
+* You have installed Red Hat Developer Hub by using either the Operator or Helm chart.
+* You have an active Redis server. For more information on setting up an external Redis server, see the link:https://www.redis.io/docs/latest/[`Redis official documentation`].
+
+.Procedure
+. Enable the Redis store cache and enter your Redis server connection URL in your `{my-app-config-file}` file:
++
+.`{my-app-config-file}` file fragment
+[source,yaml,subs="+quotes"]
+----
+backend:
+  cache:
+    store: redis
+    connection: redis://user:pass@cache.example.com:6379
+----
+
+. To enable the Redis cache for Techdocs, add the `techdocs.cache.ttl` setting in your `{my-app-config-file}` file.  This setting specifies how long, in milliseconds, a statically built asset should stay in the Redis cache.
++
+.`{my-app-config-file}` file fragment
+[source,yaml]
+----
+techdocs:
+  cache:
+    ttl: 3600000
+----
+
+. Optionally, enable the Redis cache for Technology Preview or Developer Preview plugins that have the feature, such as the Bulk Import plugin.
```

Comment:
Reading https://github.com/redhat-developer/rhdh-plugins/blob/main/workspaces/bulk-import/app-config.yaml I do not see that the Bulk Import plugin has the feature.
```suggestion
TIP: Optionally, enable also the Redis cache for unsupported plugins that have the feature. See their respective documentation.
```